contributions/epic.ts

```diff
@@ -0,0 +1,15 @@
+import { Metadata } from '../types';
```

Comment:
Realise we are calling the contributions service directly at the moment, but really this slot is not tied to contributions so it would be good not to place it in a contributions folder.

Reply:
I've moved this function to `slots.ts` at the root level, it does make sense as the slots aren't contribution specific, even though they're only used in that context right now.

But the idea was to use the main `index.ts` at the root level as the interface for the client lib so ultimately it doesn't matter how we organise the files/folders internally - no client would have to change how they import a function if we decided to move things around in the client lib, to have a folder structure that better reflects what we're doing at a given point.